resources/js/components/layouts/LayoutUser/components/TheNotification.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div class="nav-link dropdown notification">
+    <a
+      id="dropdownMenuButton"
+      class="d-flex align-items-center"
+      data-toggle="dropdown"
+    >
+      {{ $t("route.notifications") }}
+      <span class="badge badge-pill badge-info ml-2">{{ unReadNotification }}</span>
+    </a>
+    <div
```

Comment:
cho nó thêm 1 trường hợp else chị ơi.
kiểu show text: "Fucking nothing to show babe!!" 
để trong trường hợp ko có notification á chị

Reply:
đã fix